app/components/leidinggevendenbeheer/functionaris-form.js

```diff
@@ -2,6 +2,7 @@ import Component from '@glimmer/component';
 import { inject as service } from '@ember/service';
 import { tracked } from '@glimmer/tracking';
 import { action } from '@ember/object';
+import { FUNCTIONARIS_STATUS_CODE_AANGESTELD_URI } from 'frontend-lmb/utils/constants';
```

Comment:
thinking: separate file called well-known-uris? so you don't have to put _uri at the end

Reply:
or a hash with a name like that in the constants could also work